src/change.c

```diff
@@ -1500,9 +1500,11 @@ open_line(
     if (dir == FORWARD && did_ai)
 	trunc_line = TRUE;
 
+    if ((flags & OPENLINE_FORCE_INDENT) && second_line_indent >= 0)
```

Comment:
If we add `OPENLINE_FORCE_INDENT`, it should be mentioned in the docstring for open_line. But hopefully we can avoid it.